2 benchmark challenge (#9)
getml/getml-relbench: README.md

Ready to elevate your data science skills and make a real impact? Here's what awaits you:

* **Build better models faster:**  
Learn how to use getML FastProp (short for Fast Propositionalization) for automated feature extraction from relational and time-series data to deliver more accurate prediction models without manual SQL or deep business domain expertise. [getML-community](https://github.com/getml/getml-community) is open source (ELv2), so you can keep using it to turbocharge your next predictive analytics project and beyond!

* **Gain recognition and rewards:**  
Showcase your skills by surpassing benchmark scores with the help of getML, and receive €100 as a token of our gratitude for every accepted submission. Your contributions may also be featured in community spotlights and future publications!

* **Expand your network:**  
Connect with the getML dev team and other data scientists on our [Discord](https://discord.gg/B4cC9uZHdx). Share your knowledge, learn from others, and help shape the future of open-source automated feature engineering.

> > This challenge will be open for submissions starting on the **20th of January 2025 at 16:00 CET!**

Author: ```suggestion
> [!NOTE]  
> This challenge will be open for submissions starting on the **20th of January 2025 at 16:00 CET!**
```

> > We will periodically add new challenges until we run out of tasks. New challenges will be added on the **20th of January at 16:00 CET.**

Author: ```suggestion
> [!NOTE]  
> We will periodically add new challenges until we run out of tasks. New challenges will be added on the **20th of January at 16:00 CET.**
```

* We will contact you using the email address associated with your commit to gather the necessary details.  
* If you are using [GitHub’s privacy feature](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address) to keep your email private, or if you prefer us to contact you at a different email address, please follow these steps:  
   1. To ensure the reward is delivered to the correct person without requiring you to publicly share your email address, use our [encryption tool](https://getml.com/relbench/encrypt) to securely encrypt your email.
   2. Include the encrypted email address in your PR. This method allows us to maintain your privacy while verifying the recipient.

## Get Started with getML FastProp

We've prepared two example notebooks to help you get started:

* [`hm-churn.ipynb`](https://github.com/getml/getml-relbench/blob/main/hm-churn.ipynb): A classification example using the H&M dataset, showcasing how to predict customer churn

> * [`hm-item.ipynb`](https://github.com/getml/getml-relbench/blob/2-benchmark-challenge/hm-item.ipynb): A regression example using the H&M dataset, demonstrating how to forecast item sales using FastProp together with LightGBM and Optuna

Author: ```suggestion
* [`hm-item.ipynb`](https://github.com/getml/getml-relbench/blob/main/hm-item.ipynb): A regression example using the H&M dataset, demonstrating how to forecast item sales using FastProp together with LightGBM and Optuna
```
